components/Header.tsx

```
@@ -8,8 +8,10 @@ import { type Header } from "@/types";
 import PurpleIcon from "@/components/icons/PurpleIcon";
 import LighteningIcon from "@/components/icons/LighteningIcon";
 import CreateWebinarButton from "@/components/CreateWebinarButton";
+import { StripeElements } from "@/components/Stripe/StripeElements";
+import SubscriptionModal from "@/components/Stripe/SubscriptionModal";
 
-const Header = ({ user }: Header) => {
+const Header = ({ user, stripeProducts }: Header) => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Header'. Consider to delete it or rename it.

'Header' is defined here:


(lint/suspicious/noRedeclare)